modules/popup/css/lib/popup.scss

```diff
@@ -1,33 +1,37 @@
-.wdc-popup {
+@mixin wdc-popup-common() {
```

Comment:
I named the mixin `wdc-popup-common` since it seemed like that was the convention, but I can remove `-common` if you'd prefer.